README.md

```diff
@@ -1486,6 +1487,21 @@ clamp(3, false, 8); // throws
 
 ### [just-modulo](https://www.npmjs.com/package/just-modulo)
 
+:icecream:[`Try It`](https://anguscroll.com/just/just-is-prime)
+
+`npm install just-is-prime`
+
+```js
+import modulo from "just-is-prime";
+
+isPrime(1); // false
```

Comment:
can you add examples for a) non integer numbers b) other types c) null/undefined, all of which should throw
(similar to https://github.com/angus-c/just/blob/master/packages/array-median/README.md#L13)